account_vat_period_end_statement/migrations/8.0.3.0.0/post-migration.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        end_statements = env['account.vat.period.end.statement'].search([])
+        for end_statement in end_statements:
+            companies = end_statement.mapped('period_ids.company_id')
+            company_id = companies and companies[0] and companies[0].id or 1
```

Comment:
You use directly "1". What happens if i delete the strandard company and create a new one?